File: android/app/src/main/java/com/ascend/transform/MockData.kt
// MockData.kt
package com.ascend.transform

data class MockPhoto(
    val id: String,
    val url: String,
    val thumbUrl: String,
    val takenAt: String,
    val weight: Double?,
    val bodyFat: Double?,
    val journal: String?
)

val samplePhotos = listOf(
    MockPhoto("p1", "mock_before", "mock_thumb", "2026-01-01", 82.0, 24.0, "Finally committed."),
    MockPhoto("p2", "mock_after", "mock_thumb", "2026-03-01", 75.0, 18.0, "Best I've ever looked.")
)

File: ios/Ascend/Transform/MockData.swift
// MockData.swift
// Local mock data and helpers for iOS preview and development

import Foundation

struct MockPhoto: Identifiable {
    let id: String
    let url: String
    let thumbUrl: String
    let takenAt: Date
    let weight: Double?
    let bodyFat: Double?
    let journal: String?
}

let samplePhotos: [MockPhoto] = [
    MockPhoto(id: "p1", url: "mock_before", thumbUrl: "mock_thumb", takenAt: Date(), weight: 82, bodyFat: 24, journal: "Finally committed."),
    MockPhoto(id: "p2", url: "mock_after", thumbUrl: "mock_thumb", takenAt: Date(), weight: 75, bodyFat: 18, journal: "Best I've ever looked.")
]
